packages/neo-one-node-protocol/src/consensus/ConsensusContext.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+import { utils } from '@neo-one/utils';
+
+export default class ConsensusContext {
+  fastForwardSeconds: number;
+
+  constructor({ fastForwardSeconds }: {| fastForwardSeconds: number |}) {
```

Comment:
Probably just construct without an argument? I.e. `this.fastForwardSeconds = 0;`